### ISSUE TITLE
IMOS check_geospatial_x_min_max too precise

### ISSUE THREAD

**Issue by mhidas** [aodn/compliance-checker#76]
These methods use strict equality to compare the min/max of the data with the global attributes, so we get results like this:

```
globalattr                             :3:    57/61 :  
    geospatial_lat_max                 :3:     1/ 2 :  
        check_maximum_value            :3:     0/ 1 : Maximum value of LATITUDE
                                                      (-42.599998) does not
                                                      match attributes
                                                      (-42.600000)
```

Need to re-write the comparison to use the `np.isclose` function, which allows a bit of leeway.

**Comment by ggalibert** [comment]
Could you please provide a dataset example?

**Comment by ggalibert** [comment]
I suggest I modify the `check_value` function in imos/util.py so that it uses `numpy.isclose`.

**Comment by ggalibert** [comment]
See PR https://github.com/aodn/compliance-checker/pull/82

**Comment by mhidas** [comment]
@ggalibert Looks good. All it needs is a unittest to check for this issue.

**Comment by ggalibert** [comment]
PR updated.

**Comment by mhidas** [comment]
Fixed in #82
